src/routing_node.rs

```diff
@@ -921,33 +910,37 @@ impl RoutingNode {
                               _to_authority: Authority,
                               response_token: SignedToken)
                               -> RoutingResult {
-        debug!("handle ConnectRequest");
+        debug!("Handle ConnectRequest");
         match request {
             InternalRequest::Connect(connect_request) => {
                 if !connect_request.requester_fob.is_relocated() {
                     return Err(RoutingError::RejectedPublicId);
                 };
-                // first verify that the message is correctly self-signed
-                if !response_token.verify_signature(&connect_request.requester_fob
-                    .signing_public_key()) {
+                // First verify that the message is correctly self-signed.
+                if !response_token.verify_signature(
+                        &connect_request.requester_fob.signing_public_key()) {
                     return Err(RoutingError::FailedSignature);
                 };
                 if !self.core.check_node(&ConnectionName::Routing(
```

Comment:
can you add a TODO that the address relocation cache needs to be checked at this point?